frontend/src/app/AppRouter.tsx

```diff
@@ -165,7 +165,9 @@ export const AppRouter = (): JSX.Element => {
         >
           <Route index element={<CreatePage />} />
           <Route path={ADMINFORM_SETTINGS_SUBROUTE} element={<SettingsPage />}>
-            <Route path={':settingsTab'} element={<SettingsPage />} />
+            <Route path={':settingsTab'} element={<SettingsPage />}>
+              <Route path={':language'} element={<SettingsPage />} />
```

Comment:
Possible to use `frontend/src/features/admin-form/settings/SettingsPage.tsx::tabConfig` to manage the deeplink for language? -- since language seemed to be a subpath of SettingsPage

Reply:
Actually the reason I wanted a subpath with the dynamic variable of language is because I want the route `admin/form/66ba153dc06ed47840138606/settings/language/zh-SG` to match this value so that I can extract out the language value to know which list of language translations are being referenced in the view. Without this sub-route matching I am not sure of any other way to get that value at the `:settingsTab` level. This is because a route like `admin/form/66ba153dc06ed47840138606/settings/language/zh-SG` will just fall through and show the `NotFoundErrorPage` page. Do let me know if there is any other way besides this approach. Thanks!